gateway/src/apicast/policy/apicast/apicast.lua

```diff
@@ -57,8 +57,12 @@ function _M:rewrite(context)
     p:rewrite(service, context)
   end
 
+  local err
   context[self] = context[self] or {}
-  context[self].upstream = p.get_upstream(service)
+  context[self].upstream, err = p.get_upstream(service)
+  if err then
+    ngx.log(ngx.WARN, "Backend api for the service=", service.id, " is invalid, error:", err)
```

Comment:
Usually, the logs start with a lower case letter, also it is more common to use `:` rather than `=`

Also, it is not really "backend api", but rather "upstream"